lms/djangoapps/django_comment_client/forum/views.py

```diff
@@ -164,12 +153,12 @@ def get_threads(request, course_key, discussion_id=None, per_page=THREADS_PER_PA
     return threads, query_params
 
 
-def _set_group_names(course_key, threads):
+def _set_group_names(course, threads):
```

Comment:
Can you leave this variable renaming as it was? It appears to me that this is in fact a `course_key`, and that it must be a key and not a course.